### PR TITLE
Metatags config

### PR DESCRIPTION
**Pull request by CelaPablo**

### CODE REVIEW

File: src/utils/helper.js [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-  return addr?.slice(0, 6) + '...' + addr?.slice(-4)
+  if(!addr) return
+  return `${addr.slice(0, 6)} ··· ${addr.slice(-4)}`
 }
 
-export const scanLink = (chainId, addr) => {
-  return scans[chainId] + addr;
+export const scanLink = (network, addr) => {
+  const chain = chainList.filter(net => net.name.toLowerCase() === network);
+  return scans[chain[0]?.chainId] + addr;
```
Comment (maxcoto) [inline src/utils/helper.js:10]: did you make 0 (mainnet) the default? the default should be always the governance chain, arbitrum

File: src/utils/contract.js [hunk 2]
```diff
@@ -294,7 +294,9 @@ export const approveMarketMaker = async (web3, chainId, usdcAmount, walletAddres
   }
 }
 
-export const assetFetch = async (chainId, addr) => {
+export const assetFetch = async (network, addr) => {
+  const chain = chainList.filter(net => net.name.toLowerCase() === network);
+  const chainId = chain[0]?.chainId;
```
Comment (maxcoto) [inline src/utils/contract.js:299]: comment below

File: src/utils/helper.js [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
 }
 
-export const assetLink = (addr) => {
-  return "/asset/" + addr;
+export const assetLink = (addr, chainId) => {
+  const chain = chainList.filter((el) => el.chainId === Number(chainId));
+  const net = chain[0].name.toLowerCase();
+  return `/asset/${net}/${addr}`;
```
Comment (maxcoto) [inline src/utils/helper.js:16]: let's be more declarative: net => _network,  addr => _address, el => _chain

File: src/utils/helper.js [hunk 4]
```diff
@@ -1,19 +1,19 @@
...
 export const shortAddress = (addr) => {
-  return addr?.slice(0, 6) + '...' + addr?.slice(-4)
+  if(!addr) return
+  return `${addr.slice(0, 6)} ··· ${addr.slice(-4)}`
 }
 
-export const scanLink = (chainId, addr) => {
-  return scans[chainId] + addr;
+export const scanLink = (network, addr) => {
+  const chain = chainList.filter(net => net.name.toLowerCase() === network);
```
Comment (maxcoto) [inline src/utils/helper.js:9]: net => _chain

File: src/utils/contract.js [hunk 5]
```diff
@@ -294,7 +294,9 @@ export const approveMarketMaker = async (web3, chainId, usdcAmount, walletAddres
   }
 }
 
-export const assetFetch = async (chainId, addr) => {
+export const assetFetch = async (network, addr) => {
+  const chain = chainList.filter(net => net.name.toLowerCase() === network);
```
Comment (maxcoto) [inline src/utils/contract.js:298]: comment below

**Review by maxcoto — CHANGES_REQUESTED** [review]

File: src/utils/helper.js (hunk 1 above)
Comment (CelaPablo) [reply src/utils/helper.js:10]: no, I used a filter function. This return an array

Comment (maxcoto) [reply src/utils/helper.js]: You're right, then let's use `find` instead of `filter` as it will return 1 instance or null

**Pull request merged**